spack/package.py

```diff
@@ -167,10 +191,6 @@ def install(self, spec, prefix):
             for f in install_files:
                 install(os.path.join(oommfdir, f), prefix.bin)
 
-    def setup_build_environment(self, env):
-        # Seems we don't need to do anything here
-        pass
```

Comment:
Maybe we can specify `OOMMF_CPP` in here?

Reply:
Good idea. Mentioned in issue #31 .